bindings/python/iota_sdk/wallet/account.py

```diff
@@ -206,28 +207,33 @@ def unspent_outputs(self, filter_options=None):
             }
         )
 
-    def incoming_transactions(self):
+    def incoming_transactions(self) -> List[Transaction]:
         """Returns all incoming transactions of the account.
         """
-        return self._call_account_method(
+        transactions = self._call_account_method(
```

Comment:
Is this correct? Does it return it implicitly or what's happening?

Reply:
The diff looks a bit confusing, below it gets returned after converting it to a Transaction `return [Transaction.from_dict(tx) for tx in transactions]`

Reply:
Oh damn right, missed that